bin/heroku-tailscale-start.sh

```diff
@@ -39,14 +44,12 @@ else
   TAILSCALE_HOSTNAME="$TAILSCALE_HOSTNAME"
 fi
 tailscaled -cleanup > /dev/null 2>&1
-(tailscaled -verbose ${TAILSCALED_VERBOSE:--1} --tun=userspace-networking --socks5-server=localhost:1055 > /dev/null 2>&1 &)  
+(tailscaled -verbose ${TAILSCALED_VERBOSE:--1} --tun=userspace-networking --socks5-server=localhost:1055 > /dev/null 2>&1 &)
 tailscale up \
-  --authkey="${TAILSCALE_AUTH_KEY}?preauthorized=true&ephemeral=true" \
+  --authkey="${TAILSCALE_AUTH_KEY}" \
   --hostname="$TAILSCALE_HOSTNAME" \
   --advertise-tags=${TAILSCALE_ADVERTISE_TAGS:-} \
-  --accept-routes \
-  --timeout=15s \
-  ${TAILSCALE_ADDITIONAL_ARGS:---timeout=15s}
+  ${TAILSCALE_ADDITIONAL_ARGS:---accept-routes --timeout=15s}
```

Comment:
This is a refactor to make sure that the `--timeout` flag value that may be in $TAILSCALE_ADDITIONAL_ARGS is not overridden by the preexisting `--timeout=15s \` and leaves `--accept-routes` as a default but an option that could be removed as well without making a new $ENV_VAR